src/main/java/frc/robot/Vision.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Vision {
+  private double angle = 0;
+  private double distance = 0;
+  private double targetInVision = 0;
+  private double[] defaultReturn = new double[6];
```

Comment:
Nice static definition here, you've probably saved at least 1 or 2 ms of cycle time by not allocating in the loop.

Java is nice and sets these values to 0.0 for you, but in other languages, this would technically resolve to 6 values of "whatever was there" in memory, just keep that in mind.